### PR TITLE
Migrate existing inputs to vuetify

### PR DESCRIPTION
**Pull request by rossicler**
## Proposed Changes

It was migrated the text inputs of the project to vuetify. #378 

## Type of Changes

What type of change this Pull Request brings to Falko?

- [ ] Bugfix
- [x] New Feature

## Checklist

- [x] This Pull Request has a significant name.
- [ ] The build is okay (tests, code climate).
- [x] This Pull Request mentions a related issue.

### CODE REVIEW

**Comment by MatheusRich** [comment]
@rossicler It seems that you're updating all the buttons too, but it was also addressed on #379.

**Comment by rossicler** [comment]
> @rossicler It seems that you're updating all the buttons too, but it was also addressed on #379.

Thanks for the reply, I reverted the merge commit of the buttons migration.

File: src/components/Authentication/Register.vue [hunk 1]
```diff
@@ -129,12 +128,12 @@ export default {
   max-width: 20em;
 }
 
-#registerForm input {
+/* #registerForm input {
```
Comment (MatheusRich) [inline src/components/Authentication/Register.vue]: Is this still used?

File: src/components/Retrospective/AddRetrospective.vue [hunk 2]
```diff
@@ -155,14 +161,14 @@ export default {
 
 <style scoped>
 
-textarea {
+/* textarea {
```
Comment (MatheusRich) [inline src/components/Retrospective/AddRetrospective.vue]: is this still being used?

File: src/components/Retrospective/List.vue [hunk 3]
```diff
@@ -72,13 +72,13 @@ export default {
 </script>
 
 <style scoped>
-input {
+/* input {
```
Comment (MatheusRich) [inline src/components/Retrospective/List.vue]: Same here

File: src/components/Revision/ListRevision.vue [hunk 4]
```diff
@@ -66,13 +66,13 @@ export default {
 </script>
 
 <style scoped>
-input {
+/* input {
```
Comment (MatheusRich) [inline src/components/Revision/ListRevision.vue]: Same here

**Review by MatheusRich — COMMENTED** [review]
this seems to be good. Please fix the conflicts in `package.json`

**Comment by rossicler** [comment]
> this seems to be good. Please fix the conflicts in `package.json`

Conflict resolved, also deleted the commented unused css.

**Comment by MatheusRich** [comment]
@alaxalves @MartinsLucas can you guys take a look at this?

**Pull request merged**